bukuserver/middleware/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .flask_reverse_proxy_fix import ReverseProxyPrefixFix
+
+__all__ = ['ReverseProxyPrefixFix']
```

Comment:
…Apparently this file is necessary for the lint to process the files in this folder.